modin/engines/ray/pandas_on_ray/io.py

```diff
@@ -49,9 +52,15 @@ def _read_parquet_columns(path, columns, num_splits, kwargs):  # pragma: no cove
             This is used to determine the total length of the DataFrame to build a
             default Index.
     """
-    import pyarrow.parquet as pq
+ 
+    if(rowgroup>=0):
+      pf = ParquetFile(path)
+      df = pf.read_row_group(rowgroup, columns, **kwargs).to_pandas()
+      return _split_result_for_readers(0, 1, df) + [len(df.index)] 
```

Comment:
If `num_splits=1`, this will just return the `df` itself. It may not work unless you wrap it in a list yourself. We don't need to split in this case so we can just return it like this:

```python
return [df] + [len(df.index)]
```